examples/basics/scene/contour.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Simple use of SceneCanvas to display an Image.
+"""
+import sys
+
+from vispy import scene, app, visuals
+from vispy.visuals.filters import IsolineFilter
+from vispy.io import load_data_file, read_png
+
+canvas = scene.SceneCanvas(keys='interactive')
+canvas.size = 600, 800
+canvas.show()
+
+# Set up a viewbox to display the image with interactive pan/zoom
+view = canvas.central_widget.add_view()
+
+interpolation = 'bicubic'
+img_data = read_png(load_data_file('mona_lisa/mona_lisa_sm.png'))
+image = scene.visuals.Image(img_data, interpolation=interpolation,
+                            parent=view.scene)
+image.transform = visuals.transforms.STTransform(translate=(0, 0, 0.5))
```

Comment:
I don't actually need this transform for the example to run, can you remove it and the import of `visuals` above?